Makefile

```diff
@@ -0,0 +1,31 @@
+# Won't write the called command in the console
+.SILENT:
+# Because we have a folder called test we need PHONY to avoid collision
+.PHONY: test 
+
+INSTALLATION-FOLDER=./cairo
+
+install: 
+	if [ -d $(INSTALLATION-FOLDER)]; then \
+		$(MAKE) update-cairo; \
+	else \
+		$(MAKE) clone-cairo; \
+	fi
+	$(MAKE) build
+
+clone-cairo:
+	mkdir -p $(INSTALLATION-FOLDER)
+	git clone --depth 1 https://github.com/starkware-libs/cairo.git $(INSTALLATION-FOLDER)
+
+
+update-cairo:
+	git -C $(INSTALLATION-FOLDER) pull
```

Comment:
what does `-C` do? can we prefer long form commands if they're available?

Reply:
it is to avoid having to do: 
`cd cairo`
`doing the pull`
`cd ..`